### PR TITLE
Provide a way to avoid duplicated config keys, e.g., `num_classes`

### PR DESCRIPTION
**Pull request by willthefrog**

### CODE REVIEW

File: PaddleCV/PaddleDetection/ppdet/core/workspace.py [hunk 1]
```diff
@@ -144,9 +145,24 @@ def create(cls_or_name, **kwargs):
 
     kwargs = {}
     kwargs.update(global_config[name])
+
+    if getattr(config, 'shared', None):
+        for k in config.shared:
```
Comment (qingqing01) [inline PaddleCV/PaddleDetection/ppdet/core/workspace.py:151]: add some comments in code for shared

File: PaddleCV/PaddleDetection/ppdet/core/config/schema.py [hunk 2]
```diff
@@ -160,6 +161,26 @@ def validate(self):
                 self.name, ", ".join(mismatch_keys)))
 
 
+class SharedConfig(object):
+    r"""A specialized `inject` type for simple values
```
Comment (qingqing01) [inline PaddleCV/PaddleDetection/ppdet/core/config/schema.py]: Make the doc comments unified with others

File: PaddleCV/PaddleDetection/ppdet/core/workspace.py (hunk 1 above)
Comment (willthefrog) [reply PaddleCV/PaddleDetection/ppdet/core/workspace.py:151]: done

File: PaddleCV/PaddleDetection/ppdet/core/config/schema.py (hunk 2 above)
Comment (willthefrog) [reply PaddleCV/PaddleDetection/ppdet/core/config/schema.py]: done

**Review by qingqing01 — APPROVED** [review]

**Pull request merged**